mir_eval/separation.py

```diff
@@ -207,17 +209,12 @@ def bss_eval_sources(
         return (sdr[idx], sir[idx], sar[idx], popt)
     else:
         # return the default permutation for compatibility
-        popt = range(nsrc)
+        popt = list(range(nsrc))
```

Comment:
Currently (ie. on line 207 and in master) `popt` is returned as a `tuple`, not an `ndarray`. Should the documentation be changed to reflect this or should the return type be changed to match the documentation? I'd lean towards a documentation change to minimize impact on anyone currently using with function. Either choice I will change the default permutation to match this type. @craffel